src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -278,6 +288,7 @@ private String waitFor() throws InterruptedException {
             }
         }
         try {
+            this.stderr = stderr.toString(VerboseProcess.UTF_8);
```

Comment:
@diorcety instead of keeping a reference to a String variable we should keep a reference to the outputstream where the error stream is written to.